specification/cosmos-db/resource-manager/readme.md

```diff
@@ -134,14 +134,30 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
+  - repo: azure-sdk-for-net
   - repo: azure-sdk-for-python
   - repo: azure-sdk-for-java
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-node
+  - repo: azure-sdk-for-ruby
```

Comment:
pls not update this part, this is configuration for our internal CI tool, not for SDK generation.

Reply:
removed it.